src/js/components/inspectors/MultiValueProperty.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+import { resetMarkVisual } from '../../actions/markActions';
+import { PrimType } from '../../constants/primTypes';
+import { State } from '../../store';
+import { DraggingStateRecord } from '../../store/factory/Inspector';
+import { Property } from './Property';
+
+interface OwnState {
+  dragOver?: number;
+}
+
+// hunch: probably actually needs to be a bunch of different types anded to a base type
+// instead of one type with everything optional
+// but it would take a long time to figure out what the mutually exclusive fields are
+interface OwnProps {
+  primId?: number;
+  primType?: PrimType;
+  name?: string;
+  label?: string;
+  dsId?: number;
+  autoType?: 'expr' | 'tmpl';
+  onChange?: (value, arrayValues?) => void;
+  onBlur?: (value, arrayValues?) => void;
+  onKeyPress?: (value, arrayValues?) => void;
+  type?: string;
+  firstChild?: boolean;
+  droppable?: boolean;
+  opts?: string[];
+  value?: any;
+  valueProperty?: string;
+  processValue?: (value, props) => typeof value;
+  subLabel?: string;
+}
+
+interface StateProps {
+  group?: any;
+  signal?: any;
+  value?: string | number | boolean | any; // TODO: remove 'any', add Immutable.Map type
+  field?: any;
+  scale?: any;
+  srcField?: any;
+  scaleName?: any;
+  dragging?: DraggingStateRecord;
```

Comment:
For example, do you want people to be able to drag fields onto the `MultiValueProperty`? If not, let's not leave this prop here.

Reply:
Gotcha. So I am going to take this as strip it down to just what we need for now and we can add more stuff later if we need it? Gonna go with that for now.

Reply:
I removed a bunch of stuff. Lmk if I should remove more or put some stuff back.